images/vscode-ssh-server/root/init.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+set -ex
+
+# Ensure per-machine keys
+CONFIG_HOST_KEY_PATH=/config/ssh_host_ed25519_key
+# If the config doesn't have keys, generate them
+if [ ! -f $CONFIG_HOST_KEY_PATH ]; then
+    ssh-keygen -t ed25519 -N '' -q -f $CONFIG_HOST_KEY_PATH
```

Comment:
Does this print anything?  If not, maybe echo out that we're generating it for the first time.